app/vmstorage/main.go

```diff
@@ -22,10 +22,11 @@ import (
 )
 
 var (
-	retentionPeriod   = flagutil.NewDuration("retentionPeriod", 1, "Data with timestamps outside the retentionPeriod is automatically deleted")
+	retentionPeriod   = flagutil.NewDuration("retentionPeriod", "1", "Data with timestamps outside the retentionPeriod is automatically deleted")
 	snapshotAuthKey   = flag.String("snapshotAuthKey", "", "authKey, which must be passed in query string to /snapshot* pages")
 	forceMergeAuthKey = flag.String("forceMergeAuthKey", "", "authKey, which must be passed in query string to /internal/force_merge pages")
 	forceFlushAuthKey = flag.String("forceFlushAuthKey", "", "authKey, which must be passed in query string to /internal/force_flush pages")
+	snapshotsMaxAge   = flagutil.NewPromDuration("snapshotsMaxAge", "1w", "Snapshots outside the snapshotsMaxAge are automatically deleted. Check happens every hour.")
```

Comment:
It's better to disable it by default, we shouldn't change exist behavior. It may confuse users.

 WDYT about moving `snapshot autoremoval` feature into `vmbackupmanager`? 

 It can retrieve list of snapshots by API call and remove stalesnapshots by configured  maxAge.